packages/projection-typeorm/src/entity/GovernanceAction.entity.ts

```diff
@@ -0,0 +1,39 @@
+import { BigIntColumnOptions, OnDeleteCascadeRelationOptions } from './util';
+import { BlockEntity } from './Block.entity';
+import { Cardano } from '@cardano-sdk/core';
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { Hash32ByteBase16 } from '@cardano-sdk/crypto';
+
+@Entity()
+export class GovernanceActionEntity {
+  @PrimaryGeneratedColumn()
+  id?: number;
+
+  // LW-11270
+  // This is required to handle rollbacks, once we have transactions projection
+  // the OnDeleteCascadeRelationOptions can be moved on txId and this column could be removed
+  @ManyToOne(() => BlockEntity, OnDeleteCascadeRelationOptions)
+  @JoinColumn()
+  block?: BlockEntity;
+
+  @Column('varchar')
+  txId?: Cardano.TransactionId;
+
+  @Column('smallint')
+  index?: number;
+
+  @Column('varchar')
+  rewardAccount?: Cardano.RewardAccount;
+
+  @Column('varchar')
+  anchorUrl?: string;
```

Comment:
Do we need a worker job to fetch those? Is there a ticket?

Reply:
I thought for a while to the ticket to open but I wasn't able neither to choose if we need a ticket for the implementation or ATM a spike...

I propose to have first at least some discussions in one of our DS to check which targets we want to achieve and / or what we need.